com.unity.shadergraph/Editor/Data/Nodes/Utility/SubGraphNode.cs

```diff
@@ -307,6 +307,14 @@ public virtual void UpdateSlots()
                 // Copy defaults
                 switch (prop.concreteShaderValueType)
                 {
+                    case ConcreteSlotValueType.SamplerState:
+                        {
+                            var tSlot = slot as SamplerStateMaterialSlot;
```

Comment:
I'm guessing this indentation is the source of the formatting issue (might need to pull this block left one stop).